introduction.md

```diff
@@ -8,4 +8,5 @@ Deadly Skills:
 * JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?